internal/storage/bucket_handle_test.go

```diff
@@ -138,3 +139,30 @@ func (t *BucketHandleTest) TestNewReaderMethodWithInvalidGeneration() {
 	AssertNe(nil, err)
 	AssertEq(nil, rc)
 }
+
+func (t *BucketHandleTest) TestDeleteObjectMethodWithValidObject() {
+	error := t.bucketHandle.DeleteObject(context.Background(),
+		&gcs.DeleteObjectRequest{
+			Name:                       TestObjectName,
+			Generation:                 0,
+			MetaGenerationPrecondition: nil,
+		})
+
+	AssertEq(nil, error)
+}
+
+func (t *BucketHandleTest) TestDeleteObjectMethodWithMissingObject() {
+	error := t.bucketHandle.DeleteObject(context.Background(),
+		&gcs.DeleteObjectRequest{
+			Name:                       missingObjectName,
+			Generation:                 0,
+			MetaGenerationPrecondition: nil,
+		})
+
+	err_expected := errors.New("storage: object doesn't exist")
```

Comment:
We don't need to create the error, if we only comparing the string. Either we can compare directly or can keep an expected_error_message string.

Reply:
fixed